aws-google.gemspec

```diff
@@ -3,7 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'aws/google/version'
 
 Gem::Specification.new do |spec|
-  spec.required_ruby_version = '>= 3.0.5'
+  spec.required_ruby_version = '~> 3.0'
```

Comment:
I decided to loosen this requirement, rather than updating it to 3.1.7; I don't think this gem's needs are that particular. Please let me know if you disagree!